addons/website_slides/controllers/main.py

```diff
@@ -336,30 +363,43 @@ def slides_channel_home(self, **post):
             'top3_users': self._get_top3_users(),
             'challenges': challenges,
             'challenges_done': challenges_done,
-            'search_tags': request.env['slide.channel.tag']
+            'search_tags': request.env['slide.channel.tag'],
+            'slide_query_url': QueryURL('/slides/all', ['tag']),
+            'slugify_tags': self._slugify_tags,
         })
 
         return request.render('website_slides.courses_home', values)
 
-    @http.route('/slides/all', type='http', auth="public", website=True, sitemap=True)
-    def slides_channel_all(self, slide_type=None, my=False, **post):
+    @http.route(['/slides/all', '/slides/all/tag/<string:slug_tags>'], type='http', auth="public", website=True, sitemap=True)
```

Comment:
It will explode the number of combination and so the number of url and duplicate content.
What will make bot running in loop in your website until server crash.

Should be probably add same logic than blog:
tag ordered to have unique combination, don't support multi tag in get

Reply:
Seems like it is done: if get, only the first tag is kept